### PR TITLE
Add new warning message type, refactor message styling

### PR DESCRIPTION
**Pull request by gingerbreadpup**

### CODE REVIEW

File: src/sass/_vars.scss [hunk 1]
```diff
@@ -44,4 +44,4 @@ $arc_selected: #efeeed;
 $arc_row: lighten($arc_bg, 1%);
 $arc_error: #d2232a;
 $arc_success: rgba(111, 206, 119, 0.85);
-$arc_queued: rgba(245,130,31, 0.85);
+$arc_warning: rgba(245,130,31, 0.85);
```
Comment (ob6160) [inline src/sass/_vars.scss]: Param spacing so it looks nice?

**Review by ob6160 — APPROVED** [review]
One small niggle but otherwise looks good :+1:

**Review by katjam — APPROVED** [review]
Yep. Looks fine. Is there another ticket for 'Looks like you already submitted that one?' opposed to 'Has already been submitted' (presumably by mistake or fraud by someone else).

Also maybe a message for the User to 'double check' the numbers.

File: src/sass/app.scss [hunk 2]
```diff
@@ -144,22 +144,31 @@ $fa-font-path:"../node_modules/font-awesome/fonts";
 // Styling for message text
 .message {
   padding-bottom: 1em;
+  &.queue,
```
Comment (katjam) [inline src/sass/app.scss]: Just checking - is the .queue class in use?

Comment (gingerbreadpup) [reply src/sass/app.scss]: Nope, have removed it

**Comment by gingerbreadpup** [comment]
@katjam Nope, I don't think there's a ticket for that. :grimacing:

**Pull request merged**